geoSegment/geoSegment.js

```diff
@@ -240,7 +240,11 @@ async function processSessionSeriesItems(items, segments) {
     } else if (item.data['beta:affiliatedLocation'] && item.data['beta:affiliatedLocation'].geo) {
       sessionSeriesPhysicalLocationGeo = item.data['beta:affiliatedLocation'].geo;
     } else {
-      continue;
+      sessionSeriesPhysicalLocationGeo = null;
+      // if (item.data.eventAttendanceMode === "https://schema.org/OnlineEventAttendanceMode") {
```

Comment:
I agree with this idea. We should only accept locationless sessions if they have online-only attendance mode